back/node_watcher/datamodel.prisma

```diff
@@ -1,17 +1,20 @@
 type BlockNumber {
-  number: Int! @unique @id
+  id: ID! @id
+  number: Int! @unique
```

Comment:
https://github.com/prisma/prisma/issues/3233 to use Int and @id, i think we need a sequence (didn't look much into it). For simplicity we can keep it like this, or we could use their sequence command

Reply:
I have some uncommited changes to address this, blocked (as far as testing) by the workspace bug we discuess on Riot.